src/Sylius/Bundle/UiBundle/Tests/Functional/templates/templateEventsButComponent.txt.twig

```diff
@@ -0,0 +1,2 @@
+{{ sylius_template_event('template_event') }}
+{{ sylius_template_event('template_event_with_context', { context: 'The king is dead, long live the king!' }) }}
```

Comment:
Does this context change anything? I don't see the value anywhere else.

Reply:
Nope, but thanks to your comment I discovered context from the `sylius_template_event` hasn't been passed xD.